src/migrations/20200827192413-create-user.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+    id: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    hsesUserId: {
+      type: Sequelize.INTEGER,
```

Comment:
I'd make this a string, I think. I like to default to external identifier type things being strings if you're not going to be doing math-type things with them. That can save us if the hses id ever becomes a UID, for instance, and has `-`